_pages/en/capabilities.md

```diff
@@ -4,47 +4,218 @@ title: High Performing IT Capabilities
 
-IITB and ESDC must continuously improve these capabilities:
+### Lead time
 
-### Technical
+For the primary application or service you work on, what is your lead time for changes (i.e., how long does it take to go from code committed to code successfully running in production)?
 
-- Version control
-- Continuous integration
-- Deployment automation
-- Trunk-based development
-- Test automation
-- Architecture
-- Empowering teams to choose tools
-- Test data management
-- Shifting left on security
+### Deploy frequency
 
-### Process
+For the primary application or service you work on, how often does your organization deploy code to production or release it to end users?
 
-- Team experimentation
-- Streamlining change approval
-- Customer feedback
-- Visibility of work in the value stream
-- Working in small batches
+### Time to restore
 
-### Measurement
+For the primary application or service you work on, how long does it generally take to restore service when a service incident or a defect that impacts users occurs (e.g., unplanned outage, service impairment)?
 
-- Monitoring systems to inform business decisions
-- Proactive failure notification
-- Work in process limits
-- Visual management capabilities
+### Change fail percentage
 
-### Cultural
+For the primary application or service you work on, what percentage of changes to production or releases to users result in degraded service (e.g., lead to service impairment or service outage) and subsequently require remediation (e.g., require a patch, rollback, fix forward, patch)?
 
-- Job satisfaction
-- Westrum organizational culture
-- Learning culture
+### Goals
+
+High software delivery performance
+
+- Lead time [1-7] days
+- Deploy frequency [1-7] days
+- Time to restore [1-24] hours
+- Change fail percentage [0-15] %
+
+[Continuous improvement](https://cloud.google.com/solutions/devops/devops-culture-transform) of the following capabilities:
+
+- [Technical Capabilities](#1-technical-capabilities)
+  - [Version control](#11-version-control)
+  - [Continuous integration](#12-continuous-integration)
+  - Deployment automation
+  - Trunk-based development
+  - Test automation
+  - Architecture
+  - Empowering teams to choose tools
+  - Test data management
+  - Shifting left on security
+- [Process Capabilities](#2-process-capabilities)
+  - Team experimentation
+  - [Streamlining change approval](#22-streamlining-change-approval)
+  - Customer feedback
+  - Visibility of work in the value stream
+  - Working in small batches
+- [Measurement Capabilities](#3-measurement-capabilities)
+  - Monitoring systems to inform business decisions
+  - Proactive failure notification
+  - Work in process limits
+  - Visual management capabilities
+- [Cultural Capabilities](#4-cultural-capabilities)
+  - Job satisfaction
+  - [Westrum organizational culture](#42-westrum-organizational-culture)
+  - Learning culture
+
+### 1. Technical Capabilities
```

Comment:
recommendation
This could be build as a self-assessment tool where each time could self-assess without having to read the outcomes/indicators themselves (we could always expose the self-assessment tool algorithm, including those outcomes and indicators).

Ways to improve could then be one of two things:
1) link to DORA site directly  (on the specific capability page)
2) resources within ESDC (e.g. SAFER Lab + Virtual Desktop Image from the Innovation team to allow experimentation, onboarding guidance to an ESDC Code Repository (none yet but Azure Repos is being prepared), Security Champion knowledge base and contact email address (currently being developed by IT Security), list of communities to participate to (e.g. DevOps CoP), IITB DoJo, etc...

Reply:
Yes, see #1022